src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int FIGURE_COUNT = 100;
+
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Random random = new Random();
+        String randomColor = colorSupplier.getRandomColor();
+        int randomNumber = random.nextInt(FIGURE_COUNT);
+        return new Ball(randomColor, randomNumber);
```

Comment:
And as for your question, I'd prefer to make use of something like next:
```suggestion
        return new Ball(colorSupplier.getRandomColor(), random.nextInt(FIGURE_COUNT));
```

Reply:
Please take a look to the previous comment with suggested solution
` return new Ball(colorSupplier.getRandomColor(), random.nextInt(FIGURE_COUNT));`